index.html

```diff
@@ -11037,6 +11037,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				&lt;input id="startTime" type="text" aria-errormessage="msgID" aria-invalid="true" value="11:30 PM" &gt;
 				&lt;span id="msgID" aria-live="assertive"&gt;&lt;span style="visibility:visible"&gt;Invalid time: the time must be between 9:00 AM and 5:00 PM&lt;/span&gt;&lt;/span&gt;
 				</pre>
+				<p class="note"><code>aria-live="assertive"</code> is used in this example in favor of <code>aria-live="polite"</code> to ensure the error message is read before the user is done interacting with the text input. Otherwise, the user may confuse the message with that of the next text input.</p>
```

Comment:
The phrasing "in favor of" makes it sound like polite is actually favored.

I don't think it is possible to guess so precisely what might happen if polite were used instead of assertive. I would prefer we offer a more general, and potentially easier to understand, rationale for using assertive.

```suggestion
				<p class="note">This example uses <code>aria-live="assertive"</code> to indicate that assistive technologies should immediately announce the error message rather than completing other queued announcements before announcing the error message. This increases the likelihood that users are aware of the error message before they move focus out of the input.</p>
```